sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/SessionRetryOptions.java

```diff
@@ -21,6 +21,33 @@ public final class SessionRetryOptions {
         this.regionSwitchHint = regionSwitchHint;
     }
 
+    @Override
+    public String toString() {
+        return String.format(
+            "(rsh:%s)",
+            this.stringifyRegionSwitchHint()
+        );
+    }
+
+    private String stringifyRegionSwitchHint() {
```

Comment:
Nit: Just a suggestion on the function name. 

`private String regionSwitchHintToString()
`